### PR TITLE
Madness

### PR DESCRIPTION
**Pull request by terakilobyte**

### CODE REVIEW

File: src/processors/removeCommand.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+describe("some stuff", () => {
+  it("foos the bar", () => {
+    expect(true).toBeTruthy();
+  });
+});
+// :remove-end:
+console.log(bar);
+// :remove-start:
+console.log("a second snippet!")
+const realm = "ask marketing, they're still figuring it out"
```
Comment (cbush) [inline src/processors/removeCommand.test.ts:35]: May not want to commit this

**Review by cbush — APPROVED** [review]
Looks great. A few little comments then I think it's ready to merge into the feature branch.

File: src/processors/Processor.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    this.process = this.process.bind(this);
+  }
+  abstract process(command: CommandNode): CommandResult;
+}
+
+export default class Processor {
+  private static commands: Record<string, CommandProcessor> = {};
+  private static bluehawk = new Bluehawk();
+
+  static process({ commands, source }: BluehawkResult): string {
```
[the remark this answers is not in the document]
Comment (cbush) [reply src/processors/Processor.ts:32]: This does not have to be fixed now

File: src/processors/removeCommand.test.ts (hunk 1 above)
Comment (terakilobyte) [reply src/processors/removeCommand.test.ts]: gone

**Pull request merged**